sdk/python/feast/repo_config.py

```diff
@@ -420,15 +409,13 @@ def _validate_feature_server_config(cls, values):
             )
             feature_server_config_class(**values["feature_server"])
         except ValidationError as e:
-            raise ValidationError(
-                [ErrorWrapper(e, loc="feature_server")],
-                model=RepoConfig,
-            )
+            raise e
```

Comment:
no need to change this right?

Reply:
The Pydantic v2 does not have ErrorWrapper() anymore.